package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Resturant-Page",
+  "version": "1.0.0",
+  "description": "A simple web app to dislay a resturant page using HTML, JavaScript and Webpack.",
+  "main": "index.js",
+  "scripts": {
```

Comment:
- [ ] [OPTIONAL] Try to also add the 'start' script from the last reviewer example. it will require you to install the webpack-dev-server as a dependency and set its config at the webpack configuration file but it's one of the most convenient and used scripts in all Javascript frameworks.